client.go

```diff
@@ -792,6 +760,37 @@ func (c *HelmClient) rollbackRelease(spec *ChartSpec, version int) error {
 	return client.Run(spec.ReleaseName)
 }
 
+// updateDependencies checks dependencies for given helmChart and updates dependencies with metadata if dependencyUpdate is true. returns updated HelmChart
+func updateDependencies(helmChart *chart.Chart, chartPathOptions *action.ChartPathOptions, chartPath string, c *HelmClient, dependencyUpdate bool, spec *ChartSpec) (*chart.Chart, error) {
+	if req := helmChart.Metadata.Dependencies; req != nil {
+		if err := action.CheckDependencies(helmChart, req); err != nil {
+			if dependencyUpdate {
+				man := &downloader.Manager{
+					ChartPath:        chartPath,
+					Keyring:          chartPathOptions.Keyring,
+					SkipUpdate:       false,
+					Getters:          c.Providers,
+					RepositoryConfig: c.Settings.RepositoryConfig,
+					RepositoryCache:  c.Settings.RepositoryCache,
+					Out:              c.output,
+				}
+				if err := man.Update(); err != nil {
+					return nil, err
+				}
+
+				helmChart, chartPath, err = c.getChart(spec.ChartName, chartPathOptions)
```

Comment:
`chartPath` appears to be unused. As it is not returned anyway, we can ignore it here:
```suggestion
				helmChart, _, err = c.getChart(spec.ChartName, chartPathOptions)
```

Reply:
Done, thanks for the feedback @elenz97 .